src/app/profile/settings/connected-accounts/connected-accounts.spec.ts

```diff
@@ -2,8 +2,8 @@ import { Component } from '@angular/core';
 import { Contexts } from 'ngx-fabric8-wit';
 import { AuthenticationService } from 'ngx-login-client';
 import { UserService } from 'ngx-login-client';
-import { empty as observableEmpty, Observable,  of ,  throwError as observableThrowError } from 'rxjs';
-import { initContext, TestContext } from 'testing/test-context';
+import { empty as observableEmpty, of , throwError } from 'rxjs';
```

Comment:
similar to the changes in the deployments PR, should `empty as observableEmpty` simply become `empty`?

Reply:
If you do make that switch use the upper case EMPTY.